cluster/examples/kubernetes/ceph/operator.yaml

```
@@ -29,6 +29,9 @@ data:
   ROOK_CSI_ENABLE_RBD: "true"
   ROOK_CSI_ENABLE_GRPC_METRICS: "false"
 
+  # set to true to enable host networking for CSI CEPHFS and RBD nodeplugins.
```

Do we know of a scenario where it might need to be enabled? If not, let's at least mention in a comment when they might need to. What about this?
```suggestion
  # Set to true to enable host networking for CSI CEPHFS and RBD nodeplugins. This may be necessary in some network configurations where the SDN does not provide access to an external cluster.
```